packages/runtime/src/controls/rich-text/fixtures/list.ts

```diff
@@ -0,0 +1,111 @@
+import { RichTextDAO } from '../types.old'
+
+export const listFixture: RichTextDAO = {
+  object: 'value',
+  annotations: undefined,
+  data: undefined,
```

Comment:
the translation I have made ignores these properties and the `data` property on the `object: 'document'`

The construct of a Value and a Document are no longer part of slate, since I don't think we used these anyway I think it is say to not include them in the translation.